make-go.bash

```diff
@@ -44,6 +25,29 @@ case "$1" in
         exit 1
 esac
 
+RESET=1
+
+MYDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -f "$MYDIR/ext/golang/go1.4/bin/go" ]; then
+    GOROOT_BOOTSTRAP=$MYDIR/ext/golang/go1.4
+fi
+
+if [ -z "$GOROOT_BOOTSTRAP" ]; then
+    # We need Go 1.4 to bootstrap Go 1.5
+    if [ -z $GOROOT ] || [[ $(go version) != go\ version\ go1.4* ]] ; then
+            git submodule update --init ext/golang/go1.4
+            # Build Go 1.4 for host
+            pushd ext/golang/go1.4/src
+            ./make.bash --no-clean
+            popd
+            # Add Go 1.4 to the environment
+            export GOROOT="$(pwd)/ext/golang/go1.4"
+    fi
+    # Add Go 1.4 compiler to PATH
+    export GOROOT_BOOTSTRAP=$GOROOT
+fi
+
```

Comment:
It's probably because I haven't set up Go correctly on my laptop. But I'm too lazy for that right now, and as I said, it might break the F-Droid built. If you wanna change this, put it in a seperate PR please ;)